### ISSUE TITLE
Is seq2seq UNIVERSAL for all languages?

### ISSUE THREAD

**Issue by bota7070** [tensorflow/nmt#162]
Hi!
I`m from Kazakhstan. I want to run nmt for eng-kaz pairs. 
I have a question! Does this program work for eng-kaz pairs? 
I just changed train and vocabulary parts. Is it all?

**Comment by ssokhey** [comment]
Yes, It will work for any language pair as long as you have a parallel corpus. You can start with the set of standard hyperparameters available here: [standard_hparams](https://github.com/tensorflow/nmt/tree/master/nmt/standard_hparams)
